### PR TITLE
Make pylint calm down.

### PR DESCRIPTION
**Pull request by 10c8**
Just some minor changes to make pylint shut up (and increase readability a little bit).

### CODE REVIEW

File: _cloud/__init__.py [hunk 1]
```diff
@@ -9,7 +9,11 @@
 
 
 class Module(object):
-    """The magical API around downloading / installing /Â importing things."""
+    """The magical API around downloading / installing / importing things."""
+    
+    _zip = None
+    metadata = None
```
Comment (controversial) [inline _cloud/__init__.py]: Confused about where this variable is later referenced

Comment (controversial) [reply _cloud/__init__.py]: There is a metadata variable, but why does it need to be defined here?

**Review by controversial — COMMENTED** [review]
Looks overall good, I should've caught the `zip` thing.

File: _cloud/__init__.py (hunk 1 above)
Comment (10c8) [reply _cloud/__init__.py]: There are lines which define `self._zip` and `self.metadata` out of the `__init__` function (which triggers pylint for some reason).

Comment (controversial) [reply _cloud/__init__.py]: I don't think pylint is something we're developing against. I'm using pep8 and pyflakes. Let me take a look when I get to a laptop in a few hours, though.

Comment (10c8) [reply _cloud/__init__.py]: I just want to make sure I can focus at the real warnings, instead of having the editor over saturated with those silly ones.

**Review by controversial — CHANGES_REQUESTED** [review]
Can you take al the changes out besides the doctoring fix and changing `zip` to `_zip`? Also, I think something like `zipfile` would make more sense than `_zip` since it's not a private attribute. I'll merge after this. I'd just prefer not to have the top-level definitions of `_zip` and `metadata` as `None`. Thanks!

**Comment by controversial** [comment]
Pretend I said "docstring" and it wan't autocorrected there ;)

**Pull request merged**